welcome.html

```diff
@@ -2,18 +2,31 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Welcome page</title>
+  <link rel="stylesheet" href="./welcome.css">
 </head>
 <body>
-  <nav>
-    <a class="#schemedojoportal">Scheme Dojo Portal</a>
-    <a class="#home">HOME</a> 
-    <a class="#about">ABOUT</a> 
-    <a class="#news">NEWS</a> 
-    <a class="#contact">CONTACT</a>
+  <div>
+    <nav id="menu-br" class="content">
+      <div id="web-name-container">
```

Comment:
If you want to apply JavaScript to this part, please write the ` id` and `class` like this.
`<div id="web-name-container" class="web-name-container">`
`id` is used for JavaScript, and `class `is used for CSS.